math/src/main/java/com/powsybl/math/graph/UndirectedGraphListener.java

```diff
@@ -24,7 +24,11 @@
 
     void edgeAdded(int e, E obj);
 
+    void edgeBeforeRemoval(int e, E obj);
+
     void edgeRemoved(int e, E obj);
 
+    void allEdgesBeforeRemoval(Collection<E> obj);
+
```

Comment:
Please add default implementations to prevent breaking changes

Reply:
Default implementations added